### ISSUE TITLE
TransportCachingIT.testCachedTransport sometimes fails

### ISSUE THREAD

**Issue by ctubbsii** [apache/accumulo#1542]
Saw a failure of the transport cache in a recent build of master (commit 
e4b69f976c7223ff7dc3fd930462c323da0ef7ae , for reference, but the failure seems unrelated to that commit)

```
java.lang.AssertionError: Expected fourth and fifth transport to be the same instance expected same:<org.apache.accumulo.core.clientImpl.ThriftTransportPool$CachedTTransport@338c9b7> was not:<org.apache.accumulo.core.clientImpl.ThriftTransportPool$CachedTTransport@6397027e>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotSame(Assert.java:828)
	at org.junit.Assert.assertSame(Assert.java:771)
	at org.apache.accumulo.test.TransportCachingIT.testCachedTransport(TransportCachingIT.java:166)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by jzgithub1** [comment]
@ctubbsii ,  I working on this now.

**Comment by milleruntime** [comment]
This is sporadic but I did see this failure locally after a few attempts.

**Comment by Manno15** [comment]
From my testing, #1552 has fixed this issue. I went back and tested multiple commits including the current, the initial commit mentioned in this ticket, and then the commit from #1552. I was only able to reproduce this error from commits before #1552 and have not been able to reproduce it on any other commit after #1552.

**Comment by ctubbsii** [comment]
Looks like it was fixed by #1552, according to @Manno15 , so I'll close this.
